### PR TITLE
Flexible cron management

### PR DESCRIPTION
**Pull request by luizeof**
This give user to customize the default cron schedule

### CODE REVIEW

**Comment by luizeof** [comment]
Hi @Gregy do you have any concern about this kind of implementation?

**Comment by luizeof** [comment]
@dbhurley can you make some "Roadmap" about this repository?

File: apache/docker-entrypoint.sh [hunk 1]
```diff
@@ -71,6 +149,8 @@ if [ -n "$MAUTIC_CRON_DYNAMICS" ]; then
         echo "10,40 * * * *     www-data   php /var/www/html/app/console mautic:integration:fetchleads -i Dynamics > /var/log/cron.pipe 2>&1" >> /etc/cron.d/mautic
 fi
 
+echo >&2 "@reboot  www-data   [[ "$(ls -A /var/www/html/app/cache/ip_data 2>/dev/null)" ]] || php /var/www/html/app/console mautic:iplookup:download > /var/log/cron.pipe 2>&1"
```
Comment (Gregy) [inline apache/docker-entrypoint.sh]: This line doesn't make sense to me. You need to add it to the cronfile or drop the cron syntax and execute it directly from bash.

**Comment by Gregy** [comment]
Hi, thanks for the PR. The idea seems good but I would like to have all the cron variables consistent. Your PR makes integration cron jobs behave differently then the rest. It would be great if you could modify the integration jobs the same way but with some code to check their contents to see if someone tried to just enable them (1/true/or something that isn't in the cron format) or set their schedule.

**Pull request closed without merging**